.tool-versions

```diff
@@ -7,7 +7,6 @@ helm-ct 3.11.0
 kind 0.30.0
 kubectl 1.27.16 # The kubectl version depends on the K8s CI cluster version
 kustomize 5.7.1
-make 4.3
```

Comment:
Removed 'make' from the asdf, as we have experienced many network issues during installation. It's done via apt now.